lib/perl/Genome/InstrumentData/AlignmentResult.pm

```diff
@@ -1094,7 +1098,7 @@ sub _create_bam_index {
     return 1;
 }
 
-sub _create_bam_flagstat {
+sub create_bam_flagstat {
```

Comment:
Does this one also need to lock before doing its work like `create_bam_header` does?